src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala

```diff
@@ -0,0 +1,77 @@
+package cromwell.engine.backend.local
+
+import java.util.UUID
+
+import cromwell.binding.WdlExpression.ScopedLookupFunction
+import cromwell.binding.command.{StringCommandPart, Command}
+import cromwell.binding.{RuntimeAttributes, Task, Call, WorkflowDescriptor}
+import org.scalatest.{FlatSpec, Matchers}
+import org.specs2.mock.Mockito
+
+import scala.util.{Failure, Success}
+
+class LocalBackendSpec extends FlatSpec with Matchers with Mockito {
+
+  val stdoutCommand = "echo beach"
+  val stderrCommand = "echo beach >&2"
+
+  // Mock setup:
+  val mockWorkflowDescriptor: WorkflowDescriptor = mock[WorkflowDescriptor]
+  mockWorkflowDescriptor.name returns "Mocky"
+  mockWorkflowDescriptor.id returns UUID.randomUUID
+  val mockScopedLookupFunction: ScopedLookupFunction = mock[ScopedLookupFunction]
+
+
+  "LocalBackend" should "allow stdout if failOnStderr is set" in {
+    val call = mock[Call]
+    call.docker returns Option.empty[String]
+    call.failOnStderr returns true
+    call.task returns Task("mytask", Command(Seq(StringCommandPart(stdoutCommand))), Seq(), Map[String, String]())
```

Comment:
All these tests do very similar things with this call mock, it would be DRYer to pull this out to a method parameterized by only the differences.